packages/block-editor/src/components/block-variation-transforms/style.scss

```diff
@@ -0,0 +1,38 @@
+.block-editor-block-variation-transforms {
```

Comment:
It looks like a lot of hassle to style the toggle button to look like something completely different.

Do we have other components that are customized to look and behave like a select control that allows custom HTML code for items? The one that allows picking a different font size might be a better fit in terms of styling and accessibility support.

Reply:
I have checked that (`CustomSelectControl`) but would want many more additions to support better handling of content - it's not so generic for now.. I think for now it's okay and we can revisit whether to enhance the CustomSelect or create something else.

Reply:
`CustomSelectControl` is a tiny wrapper over [`downshift.js`](https://github.com/downshift-js/downshift) which isn't opinionated about HTML used:
<img width="262" alt="Screen Shot 2020-11-13 at 13 54 16" src="https://user-images.githubusercontent.com/699132/99074270-bd4b3e80-25b7-11eb-8a04-860e7e64dc10.png">

I would be very surprised if you wouldn't be able to use it here, but I agree that you might need to apply some tweaks as well. The benefit is that it better reflects its purpose from the accessibility point of view.

Reply:
I started with that, but saw that needed enough tweaks and seemed rather specific contrary to its name.
1. You think I should change it now or in a follow up PR?
2. The `DropdownMenu` suffers from accessibility point of view? Should we look how to improve this as well? (I haven't dig into accessibility yet)